src/corecel/math/Algorithms.hh

```diff
@@ -142,6 +142,30 @@ struct LogicalTrue<void>
     }
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * Evaluate whether the argument is "false".
+ */
+template<class T = void>
+struct LogicalFalse
```

Comment:
Is it a `LogicalFalse` or a `LogicalNot` (or `LogicalNegate`) ? (it look like the later, given the former I sorta assumed the implementation to be `return false;`?

Reply:
sure, I can ...

Reply:
lol... 🐿️ 

Can you open up a PR that has the find-replace plus the geant4 typo uncorrection (you probably need to add it to the codespell dictionary in scripts/dev)?

Reply:
> this should be [en.cppreference.com/w/cpp/utility/functional/logical_not.html](https://en.cppreference.com/w/cpp/utility/functional/logical_not.html)

Do you mean literally (`celeritas::LogicalFalse` -> `std::logical_not` (C++14) or in spirit -> `celeritas::LogicalNot`)?

Reply:
Hmm yeah maybe we should get rid of it and use `std::logical_not` and `thrust::logical_not`.

Reply:
I think `thrust::identity` got removed though, so we may want to make our own to be generic